src/ontology/cl-edit.owl

```diff
@@ -21529,29 +21530,32 @@ AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_0004220 "https
 AnnotationAssertion(oboInOwl:hasOBONamespace obo:CL_0004220 "cell")
 AnnotationAssertion(oboInOwl:id obo:CL_0004220 "CL:0004220")
 AnnotationAssertion(rdfs:label obo:CL_0004220 "flag amacrine cell")
-SubClassOf(obo:CL_0004220 obo:CL_0000561)
+SubClassOf(obo:CL_0004220 obo:CL_0000745)
+SubClassOf(obo:CL_0004220 obo:CL_0004251)
 
 # Class: obo:CL_0004221 (flag A amacrine cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:9620701") obo:IAO_0000115 obo:CL_0004221 "A flag amacrine cell with post-synaptic terminals in S2 and S3.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:24246263") Annotation(oboInOwl:hasDbXref "PMID:9620701") obo:IAO_0000115 obo:CL_0004221 "A flag amacrine cell with post-synaptic terminals in S2 and S3. This cell type releases the neurotransmitter glycine.")
 AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_0004221 "https://orcid.org/0000-0001-7258-9596")
 AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_0004221 "https://orcid.org/0000-0002-5260-9315")
 AnnotationAssertion(oboInOwl:hasOBONamespace obo:CL_0004221 "cell")
 AnnotationAssertion(oboInOwl:id obo:CL_0004221 "CL:0004221")
 AnnotationAssertion(rdfs:label obo:CL_0004221 "flag A amacrine cell")
 SubClassOf(obo:CL_0004221 obo:CL_0004220)
+SubClassOf(obo:CL_0004221 obo:CL_4030028)
 SubClassOf(obo:CL_0004221 ObjectSomeValuesFrom(obo:RO_0013001 obo:UBERON_0008923))
 SubClassOf(obo:CL_0004221 ObjectSomeValuesFrom(obo:RO_0013001 obo:UBERON_0008924))
 
 # Class: obo:CL_0004222 (flag B amacrine cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:9620701") obo:IAO_0000115 obo:CL_0004222 "A flag amacrine cell with post-synaptic terminals in S3, S4, and S5.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:24246263") Annotation(oboInOwl:hasDbXref "PMID:9620701") obo:IAO_0000115 obo:CL_0004222 "A flag amacrine cell with post-synaptic terminals in S3, S4, and S5. This cell type releases the neurotransmitter glycine.")
```

Comment:
Did not find evidence to support text def in dbxrefs. (flag A and flag b amacrine cell types).

Reply:
I meant the first sentence of the definition.